iina/MPVNode.swift

```diff
@@ -224,12 +224,12 @@ extension mpv_node: CustomDebugStringConvertible {
 
   /// A textual representation of this instance, suitable for debugging.
   ///
-  /// If the node is an array or a map the string will contine mulitple lines.
+  /// If the node is an array or a map the string will continue multiple lines.
```

Comment:
Oh no! This typu was added by low-batt!! It is such a bad typo that it looks like continue was misspelled, but it should be "contain", not continue.